gems/rails/CVE-2024-26143.yml

```diff
@@ -0,0 +1,46 @@
+---
+gem: rails
+cve: 2024-26143
+ghsa: 9822-6m93-xqf4
+url: https://github.com/rails/rails/security/advisories/GHSA-9822-6m93-xqf4
+title: Rails has possible XSS Vulnerability in Action Controller
+date: 2024-02-27
+description: "# Possible XSS Vulnerability in Action Controller\n\nThere is a possible
```

Comment:
Please use `description: |` so that the text isn't an escaped double-quoted String, which makes it easier to read and edit.